lib/tests/producer_accessor.cpp

```diff
@@ -500,6 +500,177 @@ void secwProducerAccessorTest(fty::SocketSyncClient& syncClient, mlm::MlmStreamC
         }
     }
 
+    // test 5.7 => insertNewDocument SNMPV3 - SHA256 / AES256
```

Comment:
You could create function or lamda to create document, do not repeat this "footcloth" of exceptions handling.

Reply:
we should probably review all the UTs and update them to catch2 style